src/NServiceBus.Core/InitializableEndpoint.cs

```diff
@@ -65,9 +65,10 @@ public InitializableEndpoint(SettingsHolder settings, IContainer container, List
 
             container.ConfigureComponent(b => settings.Get<Notifications>(), DependencyLifecycle.SingleInstance);
 
-            await RunInstallers(concreteTypes).ConfigureAwait(false);
+            var receiveInfrastructure = transportInfrastructure.ConfigureReceiveInfrastructure();
```

Comment:
This change means that `ConfigureReceiveInfrastructure` will always be called now, where before this would not have been called for a send-only endpoint. I'd like to preserve that behavior.